src/features/spa/aggregate/index.js

```diff
@@ -166,7 +167,7 @@ export class Aggregate extends AggregateBase {
     register(FN_START, function (args, eventSource) {
       var ev = args[0]
       var evName = ev.type
-      var eventNode = ev.__nrNode
+      var eventNode = ev[`__nrNode:${bundleId}`]
```

Comment:
Is there any reason to keep the `__nrNode:` piece to this?  Kinda strange syntax

Reply:
Not really but honestly I think we need to revisit this and the IDs used in function wrapper, etc. This only supports multiple bundles and does not work for multiple agents of the same bundle. I also wonder if there is a better way to maintain context data like this without modifying the object that is being passed around.